app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -66,6 +68,9 @@ class HomeFragmentPresenter @Inject constructor(
     itemList.add(allTopicsViewModel)
     topicListAdapter = TopicListAdapter(activity, itemList, promotedStoryList)
 
+    storyProgressTestHelper.markFullStoryProgressForFractions(profileId)
```

Comment:
As mentioned earlier, this is just for testing purpose so that you can actually see the content we do not this in the app. You will need to revert these changes in this file.

Reply:
Removed